README.md

```diff
@@ -1,7 +1,7 @@
 t-digest
 ========
 
-A new data structure for accurate on-line accumulation of rank-based statistics such as quantiles
+A new data structure for accurate online accumulation of rank-based statistics such as quantiles
```

Comment:
this is a matter of opinion. Different folks prefer different ways.

Happy to move to your form